Add link for Tizen.Wearable.CircularUI

jkpu opened the pull request:
### Change Description ###
Add link for Tizen.Wearable.CircularUI github at dotnet API overview

### Bugs Fixed ###
N/A

### API Changes ###
N/A

tizen-build commented:
checkout the webpage https://docs.tizen.org/staging/329

mijin85cho commented:
Target  publish date on TD : 8.7
Hello, @annie-samsung. Please take care of this PR to be merged within at least next week.

sabeena21 commented on docs/application/dotnet/api/overview.md:
```diff
@@ -24,6 +24,8 @@ The Tizen.NET contains a few limitations on the Xamarin.Forms APIs; these limita
 
 A list of limitations is available at [Current Xamarin.Forms limitations](xamarin.forms-limitations.html).
 
+You can efficiently add Tizen wearable device specific user interfaces. Extended details for these APIs are available on the [Tizen.Wearable.CircularUI github](https://github.com/Samsung/Tizen.CircularUI/).
```
Extended details for these APIs are available on the [Tizen.Wearable.CircularUI github](https://github.com/Samsung/Tizen.CircularUI/). 

Please change as:

The extended details for these APIs are available on the [Tizen.Wearable.CircularUI github](https://github.com/Samsung/Tizen.CircularUI/). 

Suggestion:
When I clicked on "https://github.com/Samsung/Tizen.CircularUI/" link it is taking me to Samsung/Tizen.CircularUI github site.

Can we change Samsung/Tizen.CircularUI  to Samsung/Tizen.Wearable.CircularUI  to avoid confusion?

jkpu commented:
I have changed the sentence for Tizen.Wearable.CircularUI follows to your review comment in your email.  and github address can't be changed. there are many projects that have different project name and github address. To avoid confusion I removed GitHub word. it means the project name, not GitHub address.



---------------------------------------------------------------------------------------------------

+You can efficiently add Tizen wearable device specific user interfaces. Extended details for these APIs are available on the [Tizen.Wearable.CircularUI github](https://github.com/Samsung/Tizen.CircularUI/).

Extended details for these APIs are available on the Tizen.Wearable.CircularUI github.

Please change as:

The extended details for these APIs are available on the Tizen.Wearable.CircularUI github.

Suggestion:
When I clicked on "https://github.com/Samsung/Tizen.CircularUI/" link it is taking me to Samsung/Tizen.CircularUI github site.

Can we change Samsung/Tizen.CircularUI to Samsung/Tizen.Wearable.CircularUI to avoid confusion?

journey2w commented on docs/application/dotnet/api/overview.md:
```diff
@@ -24,7 +24,7 @@ The Tizen.NET contains a few limitations on the Xamarin.Forms APIs; these limita
-You can efficiently add Tizen wearable device specific user interfaces. Extended details for these APIs are available on the [Tizen.Wearable.CircularUI github](https://github.com/Samsung/Tizen.CircularUI/).
+You can efficiently add Tizen wearable device specific user interfaces. The extended details for these APIs are available on the [Tizen.Wearable.CircularUI](https://github.com/Samsung/Tizen.CircularUI/).
 
 ## TizenFX API
```
@sabeena21 
Did you suggest change the original sentences except the newly added sentence based on what kinds of rules or styles?
We has already reviewed the original docs with our previous outsourcing company.

How can I know your suggestions are better than the current contents?

sabeena21 replied:
@journey2w, For the original sentences, I have not requested any major changes.
In some of the original sentences, there was missing period at the end of the sentence or alignment of some of the word such as web pages to webpages or  CRUD (Create, Read, Update, Delete) to  Create, Read, Update, Delete (CRUD) to align with our MSTP style guide
I have not requested for any major changes.

sabeena21 replied:
Please can you let me know if it okay to request some minor changes to the author?

mijin80lee commented:
TD Sync Completed.
